app/js/sagas/patientSaga.js

```diff
@@ -13,7 +13,7 @@ import actionTypes, { FETCH_PATIENT_LAB_TEST_RESULTS } from '../actions/actionTy
 import patientAction from '../actions/patientAction';
 import { selectProperty } from '../utils/globalProperty';
 
-const conceptRep = `(uuid,display,name,conceptClass:(uuid,display,name),datatype:(uuid,display,name),units)`;
```

Comment:
needed to return names here, not just the name, so that the getConceptShortName method would work